coriolis/utils.py

```diff
@@ -204,6 +204,33 @@ def write_ssh_file(ssh, remote_path, content):
     fd.close()
 
 
+@retry_on_error()
+def write_winrm_file(conn, remote_path, content):
+    """This is a poor man's scp command that transfers small
+    files, in chunks, over WinRM.
+    """
+    if conn.test_path(remote_path):
+        conn.exec_ps_command(
+            'Remove-Item -Force "%s"' % remote_path)
```

Comment:
For safety, I'd make this method check if the file exists and error out if so.
This would imply either removing the retry or moving it to the inner body. (though I'd just remove it entirely considering the `exec_ps_command` already retries intrinsically)